ape_safe/accounts.py

```diff
@@ -312,6 +364,20 @@ def call(  # type: ignore[override]
 
```

Comment:
^ having to update `call` like this:

```
        if impersonate:
            return self._impersonated_call(txn, **call_kwargs)

        call_kwargs["submit"] = call_kwargs.pop("submit_transaction", True)
        return super().call(txn, **call_kwargs)
```

Reply:
the new line is just: `call_kwargs["submit"] = call_kwargs.pop("submit_transaction", True)` tho